libs/ui/search/src/lib/ui-search.module.ts

```diff
@@ -47,6 +48,7 @@ import { UiWidgetsModule } from '@geonetwork-ui/ui/widgets'
     TagInputModule,
     UtilSharedModule,
     UiWidgetsModule,
+    FeatureCatalogModule,
```

Comment:
i d'ont think `search` depends on `catalog`, but `records` does.

Reply:
Yep, also removed this